siliconcompiler/report/streamlit_viewer.py

```diff
@@ -583,6 +584,8 @@ def show_title_and_runs(title_col_width=0.7):
     return new_chip
 
 
+ui_width = streamlit_javascript.st_javascript("window.innerWidth")
```

Comment:
Why is this in 100 lines away of where it is actually being used?

Reply:
`ui_width` is needed for this and the inclusion of the SC logo in the title. This seemed to be the most logical place to put it for both.

Reply:
lets move it closer to where it is needed here and if we decide on the SClogo then we can move it back in that PR.